### PR TITLE
Adding xyz.eclipium.Hieroctive

### PR DESCRIPTION
**Pull request by SpeedCode210**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: Hieroctive.desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Version=2.0
```
Comment (hfiguiere) [inline Hieroctive.desktop:3]:
```suggestion
```

This is not what you think it is.

File: xyz.eclipium.Hieroctive.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "xyz.eclipium.Hieroctive",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Hieroctive",
+    "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
```
Comment (hfiguiere) [inline xyz.eclipium.Hieroctive.json:10]: does it really support wayland?

File: xyz.eclipium.Hieroctive.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network"
+  	],
+    "modules": [
+        {
+            "name": "Hieroctive",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ls",
```
Comment (hfiguiere) [inline xyz.eclipium.Hieroctive.json:20]:
```suggestion
```

File: flathub.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: xyz.eclipium.Hieroctive.json (hunk 2 above)
Comment (SpeedCode210) [reply xyz.eclipium.Hieroctive.json:10]: Yes

File: flathub.json (hunk 4 above)
Comment (SpeedCode210) [reply flathub.json:2]: Unity Linux builds only supports x86_64 :/

**Comment by hfiguiere** [comment]
bot, build xyz.eclipium.Hieroctive

File: xyz.eclipium.Hieroctive.json (hunk 2 above)
Comment (nedrichards) [reply xyz.eclipium.Hieroctive.json:10]: It does indeed seem to work. Doesn't provide any window decorations if required but that's not a blocker for a game given the expected full screen context of use.

File: flathub.json (hunk 4 above)
Comment (nedrichards) [reply flathub.json:2]: Yeah, that's annoying.

**Pull request closed without merging**